### PR TITLE
TINY-6944: Update snooker to remove series column table attribute on copy

### PR DESCRIPTION
**Pull request by ltrouton**
Related Ticket: TINY-6944

Description of Changes:
* Updates snooker so that series column attribute is not included on copied table when Ctrl+C

Pre-checks:
* [x] Changelog entry added
* [X] Tests have been added (if applicable)
* ~~[ ] Branch prefixed with `feature/` for new features (if applicable)~~
* ~~[ ] License headers added on new files (if applicable)~~

Review:
* [X] Milestone set
* [x] Review comments resolved

GitHub issues (if applicable):

### CODE REVIEW

**Review by JamesToohey — APPROVED** [review]
Looks good, waiting on changelog

File: modules/snooker/src/test/ts/browser/CopySelectedTest.ts [hunk 1]
```diff
@@ -80,13 +80,20 @@ UnitTest.test('CopySelectedTest', () => {
     return table;
   };
 
-  const check = (label: string, expected: TestData[][], input: TestData[][]) => {
+  const check = (label: string, expected: TestData[][], input: TestData[][], tableAttributes?: { addBeforeCopy: Record<string, string | boolean | number>; removedAfterCopy: string[] }) => {
```
Comment (lnewson) [inline modules/snooker/src/test/ts/browser/CopySelectedTest.ts:83]: Could you split this out into a separate type for the table attributes please?

Comment (lnewson) [reply modules/snooker/src/test/ts/browser/CopySelectedTest.ts:83]: Also, `addBeforeCopy` is a bit of a confusing name. Maybe we should just call these `beforeCopy` and `afterCopy`?

**Review by lnewson — APPROVED** [review]

File: modules/snooker/src/test/ts/browser/CopySelectedTest.ts (hunk 1 above)
Comment (ltrouton) [reply modules/snooker/src/test/ts/browser/CopySelectedTest.ts]: Fixed

**Pull request merged**